anytree/iterators/__init__.py

```diff
@@ -9,9 +9,9 @@
 * :any:`ZigZagGroupIter`: iterate over tree using level-order strategy returning group for every level
 """
 
-from .abstractiter import AbstractIter  # noqa
-from .levelordergroupiter import LevelOrderGroupIter  # noqa
-from .levelorderiter import LevelOrderIter  # noqa
-from .postorderiter import PostOrderIter  # noqa
-from .preorderiter import PreOrderIter  # noqa
-from .zigzaggroupiter import ZigZagGroupIter  # noqa
+from .abstractiter import AbstractIter as AbstractIter  # noqa
```

Comment:
i guess it is for future refactoring? `as` thing. does not feels needed now?

Reply:
The `as` thing, while I admit it looks weird, is a part of telling the type checker system that we are explicitly re-exporting this particular import, because on default it assumes imports are not meant to be officially re-exported and using unofficially re-exported imports will raise cause an error to be raised by type checkers.

Reply:
https://mypy.readthedocs.io/en/latest/command_line.html#cmdoption-mypy-no-implicit-reexport